ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -452,9 +452,8 @@ const TypeConstraintMap& GetTypeConstraintMap() {
     type_constraint_map["Exp"]["T"] = NGraphNumericDTypes();
     type_constraint_map["ExpandDims"]["T"] = NGraphDTypes();
     type_constraint_map["Floor"]["T"] = NGraphNumericDTypes();
-    type_constraint_map["FloorDiv"]["T"] = NGraphNumericDTypes();
-    type_constraint_map["FloorMod"]["T"] = {
-        DT_INT32};  // IE only supports i32 for input
+    type_constraint_map["FloorDiv"]["T"] = NGraphRealDTypes();
```

Comment:
Are we restricting type due to OV-IE limitation?

Reply:
In a way, yes. If IE supported a `FloorDiv` op, we could support all numeric types here. But since it doesn't, we need to handle additional cases for integer types - especially the ones with negative integer types. This will require static input constraints on the FloorDiv op which could be added in a follow-up PR.